dotnet3.5/S3/ListObjectsExample/ListObjects/ListObjects.cs

```diff
@@ -0,0 +1,64 @@
+﻿// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX - License - Identifier: Apache - 2.0
+// snippet-start:[S3.dotnet35.ListObjects]
+
+using Amazon;
+using Amazon.S3;
+using Amazon.S3.Model;
+using System;
+using System.Threading.Tasks;
+
+namespace ListObjects
+{
+    // The following example will list the objects in an Amazon Simple Storage
```

Comment:
"The following example will list" -> "Lists"

Reply:
changed